INFO:root:gpu device = cuda:0
Traceback (most recent call last):
  File "main.py", line 278, in <module>
    save_model_str=get('run_dir'))
  File "main.py", line 214, in create_run_ensemble
    model = EnsembleModel(trained_models, batch_size=batch_size, out_size=train_dataset.n_classes)
  File "/home/rkohli/aml-project/src/model.py", line 256, in __init__
    super(MyEnsemble, self).__init__()
NameError: name 'MyEnsemble' is not defined
